sql/hive/src/main/scala/org/apache/spark/sql/hive/HiveMetastoreCatalog.scala

```diff
@@ -597,6 +619,107 @@ private[hive] class HiveMetastoreCatalog(val client: HiveClient, hive: HiveConte
     }
   }
 
+  private def convertToOrcRelation(metastoreRelation: MetastoreRelation): LogicalRelation = {
```

Comment:
Agree. It's probably OK to tolerant this duplication for this PR. I'd like to revisit this code path after finishing migrating ORC read path.